css/src/components/tag.scss

```diff
@@ -0,0 +1,92 @@
+$tag-padding-block: 0.4em !default;
+$tag-padding-inline: 0.65em !default;
+
+$tag-gap-size: 0.375em !default;
+
+$tag-font-size: $font-size-8 !default;
+$tag-font-size-sm: $font-size-9 !default;
+$tag-font-size-lg: $font-size-7 !default;
+$tag-font-weight: $weight-normal !default;
+
+$tag-icon-font-size: 0.875em !default;
+
+$tag-border-width: $border-width !default;
+$tag-border-radius: $border-radius !default;
+$tag-border-color: $secondary-dark !default;
+
+$tag-color: $secondary-dark;
+$tag-color-hover: $primary-dark;
+$tag-background: $secondary-background;
+
+$tag-color-filled: $secondary-invert;
+$tag-background-filled: $secondary;
+
+$tag-color-primary: $primary-active;
+$tag-color-primary-hover: $primary-dark;
+$tag-background-primary: $primary-background;
+
+.tag {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	padding: $tag-padding-block $tag-padding-inline;
+	border: $tag-border-width solid $tag-border-color;
```

Comment:
It looks like the default in the Fluent Figma is borderless and they have an outline variant. Do we need to match? Also, the tags currently used in Q&A seem to be borderless too.

Reply:
I want to keep outline as default to align with other components. For filled variants we normally use the matching class and I don't want to mess with this pattern.

Reply:
Should there at least be a borderless variant then?

Reply:
`border-none` atomic should do that. But wondering where do you see the mismatch?

Reply:
In the fluent v2 Figma and the Fluent v2 site, it looks to me that borderless is their default and outline/filled is a variant

![image](https://github.com/user-attachments/assets/d792777b-d23f-41d2-a8bf-4091efaa3132)

https://fluent2.microsoft.design/components/web/react/tag/usage